custom/icds_reports/ucr/data_sources/person_cases_v3.json

```diff
@@ -307,6 +307,248 @@
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "time_birth",
+        "datatype": "string",
+        "type": "raw",
+        "property_name": "time_birth"
+      },
+      {
+        "column_id": "father_name",
+        "datatype": "string",
+        "type": "raw",
+        "property_name": "father_name"
+      },
+      {
+        "column_id": "mother_name",
+        "datatype": "string",
+        "type": "raw",
+        "property_name": "mother_name"
+      },
+      {
+        "column_id": "private_admit",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_name",
+            "property_name": "private_admit"
+          },
+          "cases": {
+            "no": {
+              "type": "constant",
+              "constant": 0
+            },
+            "yes": {
+              "type": "constant",
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "date_last_private_admit",
+        "datatype": "date",
+        "type": "raw",
+        "property_name": "date_last_private_admit"
+      },
+      {
+        "column_id": "date_return_private",
+        "datatype": "date",
+        "type": "raw",
+        "property_name": "date_return_private"
+      },
+      {
+        "column_id": "primary_admit",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_name",
+            "property_name": "primary_admit"
+          },
+          "cases": {
+            "no": {
+              "type": "constant",
+              "constant": 0
+            },
+            "yes": {
+              "type": "constant",
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "is_oos",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_name",
+            "property_name": "is_oos"
+          },
+          "cases": {
+            "no": {
+              "type": "constant",
+              "constant": 0
+            },
+            "yes": {
+              "type": "constant",
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "last_class_attended_ever",
```

Comment:
what is this indicator?

Reply:
this "LastClassAttended" , Below is the comment on this in the app.
```
The last class attended by the girl: If first time registration (was_oos_ever_loaded!=yes) then take from register_ag group ; if re-registration then take from re-register_ag group ; else check what is autoloaded.

```

Reply:
yeah, He confirmed we can use ccs one. removing ths.